test/others/criu-coredump/test.sh

```diff
@@ -9,11 +9,12 @@ function _exit {
 
 function gen_imgs {
 	setsid ./loop.sh < /dev/null &> /dev/null &
-	PID=$!
-	$CRIU dump -v4 -o dump.log -D ./ -t $PID
-	if [ $? -ne 0 ]; then
-		kill -9 $PID
-		_exit 1
+	sleep .1  # wait for loop.sh to start
+	PID=$(pgrep -n loop.sh)
+	if ! $CRIU dump -v4 -o dump.log -D ./ -t "$PID"; then
+		cat dump.log
```

Comment:
Printing full log may be a bit too noizy, isn't it? You can probably grep Error or print the tail of the file instead.

Reply:
We don't expect `criu dump` to fail and most likely this was the reason why the logs were not shown when it fails. However, when the test fails in CI environment it would be difficult to find out why without the logs.

> You can probably grep Error or print the tail of the file instead.

Being able to see the whole content of `dump.log` is more useful.

Reply:
@Snorch we print it only when criu fails, so it should be fine. The verbosity has to be 4.

Reply:
Ok, sounds reasonable.